packages/playwright/.eslintignore

```diff
@@ -3,4 +3,5 @@ storybook-static/**/*.js
 
 tsup.config.ts
 playwright.config.ts
+src/inBrowserPostProcessing.js
```

Comment:
I’m not a fan of opting this entire file out of linting. I rather use disable specific rules within the file. What rules are ESLint telling about?

Reply:
Since this file is run in the browser (but all the rest of our code in the repo runs in Node.js), there are lots of linting errors for things like `define`, `require`, etc that either don't exist in node or exist in a different context. I had a couple different PR checks fail (`lint` and `build`) and didn't see a great way around it.